db/column_family.cc

```diff
@@ -347,8 +349,9 @@ ColumnFamilyData::ColumnFamilyData(
       refs_(0),
       dropped_(false),
       internal_comparator_(cf_options.comparator),
-      initial_cf_options_(SanitizeOptions(db_options, cf_options)),
-      ioptions_(db_options, initial_cf_options_),
+      initial_cf_options_(SanitizeOptions(immutable_db_options, cf_options)),
+      ioptions_(immutable_db_options, initial_cf_options_),
+      mutable_db_options_(mutable_db_options),
```

Comment:
We shouldn't store mutable_db_options in ColumnFamilyData, as the content is subject to change.